astrocut/cube_cut.py

```diff
@@ -272,17 +272,26 @@ def _fit_cutout_wcs(self, cutout_wcs, cutout_shape):
         """
 
         # Getting matched pixel, world coordinate pairs
-        pix_inds = np.array(list(product(list(range(cutout_shape[1])), list(range(cutout_shape[0])))))
+        # We don't want more than ~100 pairs, and we want them spread evenly through the iamge
+
+        y, x = cutout_shape
+        i = 1
+        while (x/i)*(y/i) > 100:
+            i += 1
+            
+        pix_inds = np.array(list(product(list(range(0,x,i)), list(range(0,y,i)))))
         world_pix = SkyCoord(cutout_wcs.all_pix2world(pix_inds, 0), unit='deg')
 
         # Getting the fit WCS
         linear_wcs = fit_wcs_from_points([pix_inds[:, 0], pix_inds[:, 1]], world_pix, proj_point=self.center_coord)
 
         self.cutout_wcs = linear_wcs
 
-        # Checking the fit
+        # Checking the fit (we want to use all of the pixels for this)
```

Comment:
What is being checked here?  Is "dists" used anywhere else?

Reply:
Nope, dists is the angular distance between the true skycoords as calculated with the original wcs and the approximate skycoords as calculated with the linear wcs.

Reply:
This is meant to be a measure of how good the approximate wcs is. It's very good for small cutouts and gets progressively worse for larger cutouts.